ace-setup-mq/prep-mq.sh

```diff
@@ -2,61 +2,104 @@
 #Functions
 # 0: print usage
 printUsage(){
-    echo -e "\033[1;33mThis script will try to cover following things:\033[0m"
+    echo "Description: "
```

Comment:
**question**: why we removed the \033 for many of the places?

Reply:
With feedback from Ben, he just wants to highlight the important information after all checking, so we don't need to mark all other log msg as green/red color.